Spec/Tests/RestClientTests.swift

```diff
@@ -1533,46 +1533,32 @@ class RestClientTests: XCTestCase {
         testStoresSuccessfulFallbackHostAsDefaultHost(.hostInternalError(code: 501))
     }
 
-    func beforeEach__RestClient__Host_Fallback__should_store_successful_fallback_host_as_default_host__should_restore_default_primary_host_after_fallbackRetryTimeout_expired() {
-        ARTDefault.setFallbackRetryTimeout(1.0)
-    }
-
     func test__080__RestClient__Host_Fallback__should_store_successful_fallback_host_as_default_host__should_restore_default_primary_host_after_fallbackRetryTimeout_expired___hostUnreachable() {
-        beforeEach__RestClient__Host_Fallback__should_store_successful_fallback_host_as_default_host__should_restore_default_primary_host_after_fallbackRetryTimeout_expired()
```

Comment:
If you're going to remove this `beforeEach`, don't its contents need to go inside `testRestoresDefaultPrimaryHostAfterTimeoutExpires`, similarly to what you've done for `testUsesAnotherFallbackHost`?

Reply:
I did, look at line 83 - `options.fallbackRetryTimeout = 1`